src/Event/Event.styles.js

```diff
@@ -6,7 +6,7 @@ const styles = StyleSheet.create({
     position: 'absolute',
     borderRadius: 0,
     flex: 1,
-    overflow: 'hidden',
+    // overflow: 'hidden',
```

Comment:
shall we remove this comment?

Reply:
wait, but if we remove it, won't the event have any problem?

Reply:
I think there is no problem. It was there to avoid overflowing the text into the grid, but that does not occur anymore (I think since the usage of Animated.View, instead of View):

https://user-images.githubusercontent.com/20805451/179537535-dd14951a-041b-4b99-ad8d-3718ffff42d9.mp4